src/machine/machine-varlink.c

```diff
@@ -825,3 +826,98 @@ int vl_method_bind_mount(sd_varlink *link, sd_json_variant *parameters, sd_varli
 
         return sd_varlink_reply(link, NULL);
 }
+
+typedef struct MachineCopyParameters {
+        const char *name;
+        PidRef pidref;
+        char *src;
+        char *dest;
```

Comment:
if you'd switch to json_dispach_const_path() as per disucssion below these could become "const char*"